### DISCUSSION TITLE
How to restore my old .history to new install on Win11?

### DISCUSSION THREAD

**Discussion by jkr4m3r** [chrisant996/clink#384] [General]
On the tin:  How to restore my old .history to new install on Win11?

**Comment by chrisant996** [comment]
I think you may be looking for this section of the documentation:

[Migrating between different profile directories](https://chrisant996.github.io/clink/clink.html#migrating-between-different-profile-directories) in the [Upgrading from Clink v0.4.9](https://chrisant996.github.io/clink/clink.html#upgradefrom049) section.

Or if you're asking about newer versions of Clink, then the file is no longer .history.

Are you asking how to copy the profile directory?

Sorry, I'm not sure exactly what question is being asked.

**Comment by jkr4m3r** [comment]
Yea, an old .history from clink 0.4.9. Thanks for the link, my aged eyes missed that.
